code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -395,7 +395,7 @@
 	icon_state = "hardsuit0-syndielite"
 	hardsuit_type = "syndielite"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/syndi/elite
-	armor = list(MELEE = 60, BULLET = 60, LASER = 50, ENERGY = 25, BOMB = 90, BIO = 100, RAD = 70, FIRE = 100, ACID = 100, WOUND = 25)
+	armor = list(MELEE = 60, BULLET = 60, LASER = 50, ENERGY = 25, BOMB = 1000, BIO = 100, RAD = 70, FIRE = 100, ACID = 100, WOUND = 25)
```

Comment:
here too, maybe make this one have 35 wound resist too to also improve the balance